optimizely/project_config.py

```diff
@@ -56,6 +57,7 @@ def __init__(self, datafile, logger, error_handler):
         self.environment_key = config.get('environmentKey', None)
         self.groups = config.get('groups', [])
         self.experiments = config.get('experiments', [])
+        self.mapexperiment = copy.deepcopy(self.experiments)
```

Comment:
this doesn't look ok to me.

Reply:
Found some other issues as well with test cases. working through now.